config/doctrine/User.orm.yml

```diff
@@ -5,35 +5,32 @@ App\Entity\User:
     id:
         id:
             type: integer
-            scale: 0
             length: null
             nullable: false
-            precision: 0
             id: true
             generator:
                 strategy: AUTO
     fields:
         username:
             type: string
-            scale: 0
             length: 255
             unique: true
             nullable: false
-            precision: 0
         password:
             type: string
-            scale: 0
             length: 255
             unique: false
             nullable: false
-            precision: 0
         roles:
             type: json_array
-            scale: 0
             length: null
             unique: false
             nullable: false
-            precision: 0
+        groupId:
+            type: string
+            length: 255
+            unique: false
```

Comment:
Just to check, as we still don't have full info on groups:
should it really be nullable?

Reply:
Yes, because for example Invalsi doesn't use groups. We'll implement a load balancing strategy (in the next pr) on top of the group, but other than that, it's not a must have column.

Reply:
Aw, yeah, indeed, forgot that roster is not client-specific.